From fb186cf9d1360193f7150aa43193fc7b0ad14029 Mon Sep 17 00:00:00 2001
From: Sridharan S N <quic_sridsn@quicinc.com>
Date: Wed, 6 Dec 2023 22:45:06 +0530
Subject: [PATCH 418/500] arm64: disable KASAN for vmalloc region

Enabling KASAN for vmalloc regions leads to high memory usage. So disable it.

With this config disabled, around 70MB consumption is reduced.

Change-Id: I294d79bf5879167334eff6a1806e6948e64ac7ae
Signed-off-by: Kathiravan Thirumoorthy <quic_kathirav@quicinc.com>
Signed-off-by: Sridharan S N <quic_sridsn@quicinc.com>
---
 arch/arm64/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/Kconfig b/arch/arm64/Kconfig
index 6062a52a084f..c58ea5d6db1f 100644
--- a/arch/arm64/Kconfig
+++ b/arch/arm64/Kconfig
@@ -235,7 +235,6 @@ config ARM64
 	select HOTPLUG_CORE_SYNC_DEAD if HOTPLUG_CPU
 	select IRQ_DOMAIN
 	select IRQ_FORCED_THREADING
-	select KASAN_VMALLOC if KASAN
 	select LOCK_MM_AND_FIND_VMA
 	select MODULES_USE_ELF_RELA
 	select NEED_DMA_MAP_STATE
-- 
2.34.1

